package.json

```diff
@@ -15,18 +15,18 @@
     "lint": "tslint --project tsconfig.json",
     "lint-fix": "tslint --project tsconfig.json --fix && echo \"👍 Lint passed\" || echo \"👎 Found few lint errors\"",
     "zapier-validate": "yarn build && zapier validate",
-    "zapier-test": "yarn build && zapier test",
+    "zapier-test": "yarn build && zapier validate && yarn test",
```

Comment:
I call jest directly here (instead of `zapier test`), because `zapier test` starting from version 9.x passes `--timeout` command line argument which is not recognizable by `jest`
According to docs, `zapier test` is the same as `npm test`
https://platform.zapier.com/cli_docs/docs#running-unit-tests